docs/UserGuide.adoc

```diff
@@ -341,7 +342,100 @@ Format: `exit`
 Address book data are saved in the hard disk automatically after any command that changes the data. +
 There is no need to save manually.
 
-== Features (Coming in v2.0)
+== Features(Coming in v2.0)
+
+=== Add a relationship
+
+Adds a relationship between two persons. +
+Format: `addRelationship INDEXOFSOURCE INDEXOFDESTINATION DIRECTION`
+
+****
+* Add a relationship between the person with `INDEXOFSOURCE` and `INDEXOFDESTINATION`, +
+with the direction of the relationship specified.
+* `INDEXOFSOURCE` refers to the index of the person from whom the relationship is +
+initiated as shown in the most recent listing.
```

Comment:
How about adding a point before this that goes, "All indexes refer to the indexes shown in the most recent listing."

So that the `SOURCE_INDEX` can be `SOURCE_INDEX` refers to the index of the person from whom the relationship is initiated (i.e. phrased similarly to `DESTINATION_INDEX` but with greater clarity).